presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -313,6 +313,33 @@ public void testPow()
         assertFunction("pow(NULL, NULL)", DOUBLE, null);
         assertFunction("pow(5.0, NULL)", DOUBLE, null);
         assertFunction("pow(NULL, 5.0)", DOUBLE, null);
+
```

Comment:
I would extract private testPower(String methodAlias) 
and call it twice:
```java
testPower("pow");
testPower("power");
```
This way you will avoid copy pasting two blocks which differ only name of called scalar.